src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -247,6 +262,24 @@ function findLink (target) {
       .favourited {
         color: #e6a23c;
       }
+
+      .toot-menu{
+        padding: 0;
+        font-size: 0.8em;
+        margin-left: 0.5em;
+        list-style-type: none;
+        text-align: center;
+
+        li{
+          padding-bottom: 0.5em;
+          border-bottom: 1px solid #ddd;
```

Comment:
I want the cursor to be a pointer when a mouse points over this object. What do you think?

Reply:
Yes, I think so too. I added the toot menu but forgot to change the cursor to pointer, sorry : (

Reply:
New commit added &:hover{ cursor: pointer; } to toot menu item.

Reply:
Thanks!